lib/std/fmt.zig

```diff
@@ -314,17 +314,27 @@ pub const ParseIntError = error{
 ///
 /// Ignores '_' character in `buf`.
 /// See also `parseUnsigned`.
+///
+/// Asserts 'T' is an integer type
```

Comment:
Generally we use the word "assert" to mean a runtime check if the function is called at runtime: https://ziglang.org/documentation/master/#Doc-Comment-Guidance